### DISCUSSION TITLE
Zigbee commands to control lidl / SilverCrest Power Strip 3 AC (16A) with 4 USB

### DISCUSSION THREAD

**Discussion by rhk** [arendst/Tasmota#16779] [Q&A] [answered]
Hi there,

I run zbbridge-tasmota 9.2.0 on Sonoff's Zigbee gateway
Program Version | 9.2.0(zbbridge)
2020-12-21T15:06:32
2_7_4_9/2.2.2-dev(38a443e)

I have been able to connect various Sonoff zb sensors (PIR, temperature), Ikea Tradfri switches as well as a Lidl/SilverCrest plug. They all work fine.

Now I bought an LIDL / SilverCrest Power Strip 3 AC (16A) with 4 USB.
It has 3 power outlets that can be controlled separately over zigbee. USB:s are not controllable, I believe (but haven't tried it yet)

These pages show that Zigbee support shouldn't be an issue. The zigbee2mqtt even shows some commands to use to control:
https://zigbee.blakadder.com/Lidl_HG06338.html
https://www.zigbee2mqtt.io/devices/HG06338.html

I was able connect and rename it with no problems:
![image](https://user-images.githubusercontent.com/404543/194838320-5972c7df-1f6c-48c2-9d8b-ddbf73f75552.png)

So far I have been able to control one of the three outlets:
loka 10 12:34:50 sonoffs/cmnd/zbbridge/ZbSend {"Device":"0xC6F6", "Send":{"Power":"ON"}}
loka 10 12:34:50 sonoffs/stat/zbbridge/RESULT {"ZbSend":"Done"}
loka 10 12:34:50 sonoffs/tele/zbbridge/RESULT {"ZbResponse":{"Device":"0xC6F6","Name":"ZbJatko1","Command":"0006!01","Status":0,**"StatusMessage":"SUCCESS"**,"Endpoint":1,"LinkQuality":29}}

However, trying to control outlets #2 and #3 dont work as I expect.. 
loka 10 12:34:59 sonoffs/cmnd/zbbridge/ZbSend {"Device":"0xC6F6", "Send":{"Power1":"ON"}}
loka 10 12:34:59 sonoffs/stat/zbbridge/RESULT Unrecognized zigbee command: Power1

loka 10 12:35:23 sonoffs/cmnd/zbbridge/ZbSend {"Device":"0xC6F6", "Send":{"Power2":"ON"}}
loka 10 12:35:23 sonoffs/stat/zbbridge/RESULT Unrecognized zigbee command: Power2

loka 10 12:35:32 sonoffs/cmnd/zbbridge/ZbSend {"Device":"0xC6F6", "Send":{"state_l1":"ON"}}
loka 10 12:35:32 sonoffs/stat/zbbridge/RESULT Unrecognized zigbee command: state_l1

loka 10 12:35:40 sonoffs/cmnd/zbbridge/ZbSend {"Device":"0xC6F6", "Send":{"state_l2":"ON"}}
loka 10 12:35:40 sonoffs/stat/zbbridge/RESULT Unrecognized zigbee command: state_l2

I'd like to get some support on how to control this / how to debug more?

Does this help:
loka 10 12:35:54 sonoffs/cmnd/zbbridge/ZbSend {"Device":"0xC6F6", "Send":{"POWER"}}
loka 10 12:35:54 sonoffs/stat/zbbridge/RESULT {"ZbSend":"Done"}
loka 10 12:35:54 sonoffs/tele/zbbridge/RESULT {"ZbResponse":{"Device":"0xC6F6","Name":"ZbJatko1","Command":"EF00!00","Status":195,"StatusMessage":"UNSUPPORTED_CLUSTER","Endpoint":1,"LinkQuality":24}}

Thanks,

r

This is my second Zb device I run into problems finding the relevant commands, also see #16449  - Is there a way for an average user to figure out the relevant commands without asking?

**Comment by rhk** [comment, marked as answer]
.. and aswering to myself finding the solution here:
https://github.com/dresden-elektronik/deconz-rest-plugin/pull/3838

Using endpoints 0x01, 0x02, 0x03 makes it possible to control outlets 1-3:

loka 10 12:47:46 sonoffs/cmnd/zbbridge/ZbSend {"Device":"ZbJatko1", "endpoint":"0x02","Send":{"Power": "0"}}
loka 10 12:47:46 sonoffs/stat/zbbridge/RESULT {"ZbSend":"Done"}
loka 10 12:47:46 sonoffs/tele/zbbridge/RESULT {"ZbResponse":{"Device":"0xC6F6","Name":"ZbJatko1","Command":"0006!00","Status":0,"StatusMessage":"SUCCESS","Endpoint":2,"LinkQuality":29}}

loka 10 12:47:53 sonoffs/cmnd/zbbridge/ZbSend {"Device":"ZbJatko1", "endpoint":"0x03","Send":{"Power": "0"}}
loka 10 12:47:53 sonoffs/stat/zbbridge/RESULT {"ZbSend":"Done"}
loka 10 12:47:53 sonoffs/tele/zbbridge/RESULT {"ZbResponse":{"Device":"0xC6F6","Name":"ZbJatko1","Command":"0006!00","Status":0,"StatusMessage":"SUCCESS","Endpoint":3,"LinkQuality":29}}

loka 10 12:47:57 sonoffs/cmnd/zbbridge/ZbSend {"Device":"ZbJatko1", "endpoint":"0x01","Send":{"Power": "0"}}
loka 10 12:47:57 sonoffs/stat/zbbridge/RESULT {"ZbSend":"Done"}
loka 10 12:47:58 sonoffs/tele/zbbridge/RESULT {"ZbResponse":{"Device":"0xC6F6","Name":"ZbJatko1","Command":"0006!00","Status":0,"StatusMessage":"SUCCESS","Endpoint":1,"LinkQuality":29}}

loka 10 12:48:02 sonoffs/cmnd/zbbridge/ZbSend {"Device":"ZbJatko1", "endpoint":"0x01","Send":{"Power": "1"}}
loka 10 12:48:02 sonoffs/stat/zbbridge/RESULT {"ZbSend":"Done"}
loka 10 12:48:02 sonoffs/tele/zbbridge/RESULT {"ZbResponse":{"Device":"0xC6F6","Name":"ZbJatko1","Command":"0006!01","Status":0,"StatusMessage":"SUCCESS","Endpoint":1,"LinkQuality":29}}

loka 10 12:48:06 sonoffs/cmnd/zbbridge/ZbSend {"Device":"ZbJatko1", "endpoint":"0x02","Send":{"Power": "1"}}
loka 10 12:48:06 sonoffs/stat/zbbridge/RESULT {"ZbSend":"Done"}
loka 10 12:48:06 sonoffs/tele/zbbridge/RESULT {"ZbResponse":{"Device":"0xC6F6","Name":"ZbJatko1","Command":"0006!01","Status":0,"StatusMessage":"SUCCESS","Endpoint":2,"LinkQuality":29}}

loka 10 12:48:12 sonoffs/cmnd/zbbridge/ZbSend {"Device":"ZbJatko1", "endpoint":"0x03","Send":{"Power": "1"}}
loka 10 12:48:12 sonoffs/stat/zbbridge/RESULT {"ZbSend":"Done"}
loka 10 12:48:12 sonoffs/tele/zbbridge/RESULT {"ZbResponse":{"Device":"0xC6F6","Name":"ZbJatko1","Command":"0006!01","Status":0,"StatusMessage":"SUCCESS","Endpoint":3,"LinkQuality":26}}

**Reply by sfromis** [reply]
Did you see the setting `ZbEndpointSuffix`/`SetOption101` ?
> Add Zigbee source endpoint as suffix to attributes ... e.g. `Power3` instead of `Power` if sent from endpoint 3.
